### PR TITLE
Clarify that some parts are directional and not functional yet

### PR DESCRIPTION
**Pull request by OceanOak**

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -1971,6 +1975,11 @@ video {
     font-size: 1.25rem;
     line-height: 1.75rem;
   }
+
+  .md\:text-lg{
```
Comment (StachuDotNet) [inline public/style.css:1979]: is this a typo or css thing I don't know?

Comment (StachuDotNet) [inline public/style.css:1979]:
```suggestion
  .md:text-lg{
```
or maybe this is some CSS thing I don't know?

Comment (OceanOak) [reply public/style.css:1979]: This file is generated from tailwind

Comment (StachuDotNet) [reply public/style.css:1979]: neat, must be some css thing I'm ignorant of :)

**Pull request merged**